src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs

```diff
@@ -370,6 +380,7 @@ public static ICustomMarshaler GetInstance(string cookie)
 
         public static void Parameter_MarshalerOnValueType_ThrowsMarshalDirectiveException()
         {
+            // typeof(System.Exception): Method '[ICustomMarshaler_TargetWindows]System.Runtime.InteropServices.Tests.ICustomMarshalerTests.MarshalerOnValueTypeMethod(int32)' requires marshalling that is not yet supported by this compiler.
```

Comment:
Jeremy is fixing this one and below in #63320 (we will get rid of "requires marshalling that is not yet supported" and just throw MarshalDirective).